examples/settings.rs

```diff
@@ -0,0 +1,48 @@
+use meilisearch_sdk::client::Client;
+use meilisearch_sdk::indexes::Index;
+use meilisearch_sdk::settings::Settings;
+
+// we need an async runtime
+#[tokio::main(flavor = "current_thread")]
+async fn main() {
+    let client = Client::new("http://localhost:7700", "masterKey");
+
+    // We try to create an index called `movies` with a primary_key of `movie_id`.
+    let my_index: Index = client
```

Comment:
In line 8, you don't define the "type" but here you defined it, is this a convention or just a non-required thing?

Reply:
Oh yes, that's useless!
I copy-pasted some code and forgot to remove it.

But now you talk about it, I think it would be good to show all the types we're using, so when someone looks at the example, he knows instantaneously which function returns what without even looking at the doc.

I'll update my PR thanks :+1:

Reply:
Awesome! This is something I need to be used to, in some communities like in the C# they are going to the full `var` style (without explicit types). Btw there is a widely used decision regarding the theme?

> he knows instantaneously which function returns what without even looking at the doc.

I like this! 👍

Reply:
Yep usually in rust you elide all the types you can.
If the compiler is able to infer something let it do it.
Sometimes you even write « half » a type like that:
```rust
let chars: Vec<_> = "a string".chars().collect();
```
Here rust can't guess which kind of data structure you want to create with the `.collect` method so you need to say it's a `Vec`, but then you can elide the `char` because the `.chars()` method returns an iterator of `char` :grin: